system_setup/server/controllers/wslconfbase.py

```diff
@@ -43,6 +47,32 @@ class WSLConfigurationBaseController(SubiquityController):
         'additionalProperties': False,
         }
 
+    def __init__(self, app):
+        super().__init__(app)
+
+        # load the config file
+        data = {}
+
+        if path.exists('/etc/wsl.conf'):
```

Comment:
Looks similar to the construct in wsl_advanced, can we refactor out a utility function?

Reply:
Gotcha. but the PR is now moved to the other one now.